float32 x
float32 y
string color	#color of cone, 'b' = blue, 'y' = yellow, 'r' = red
